sqllogictest-bin/src/main.rs

```diff
@@ -527,13 +511,16 @@ async fn run_parallel(
     db.shutdown().await;
 
     if !failed_cases.is_empty() {
-        Err(anyhow!("some test case failed:\n{:#?}", failed_cases))
+        Err(anyhow!("some test cases failed:\n{:#?}", failed_cases))
+    } else if cancel.is_cancelled() {
+        Err(anyhow!("some test cases skipped or cancelled"))
     } else {
         Ok(())
     }
 }
 
 // Run test one be one
+#[allow(clippy::too_many_arguments)]
```

Comment:
Should we introduce a ctx struct or sth?

Reply:
Next time for sure!